### PR TITLE
feat(Checkbox): improve a11y with outline

### PR DESCRIPTION
**Pull request by jsomsanith**
**What is the problem this PR is trying to solve?**
There is no outline on Checkboxes and Radios, especially when they are checked.

**What is the chosen solution to this problem?**
Add an outiline from checkbox guideline

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/theme/src/theme/_forms.scss [hunk 1]
```diff
@@ -6,6 +6,11 @@
 $form-shadow: inset 0 -1px 0 $gallery;
 $form-focus-shadow: inset 0 -2px 0 $scooter;
 
+@mixin checkOutline() {
+	outline: 2px auto -webkit-focus-ring-color;
```
Comment (annilsson) [inline packages/theme/src/theme/_forms.scss]: What about non-webkit browsers? Could we have a fallback color for them on the line before maybe?

File: packages/theme/src/theme/_forms.scss [hunk 2]
```diff
@@ -6,6 +6,11 @@
 $form-shadow: inset 0 -1px 0 $gallery;
 $form-focus-shadow: inset 0 -2px 0 $scooter;
 
+@mixin checkOutline() {
```
Comment (annilsson) [inline packages/theme/src/theme/_forms.scss]: The linter complained about camelCase, but while at it, why not checkbox-outline?

File: packages/theme/src/theme/_forms.scss (hunk 1 above)
Comment (jmfrancois) [reply packages/theme/src/theme/_forms.scss]: we still use autoprefix for compatibility

Comment (jsomsanith) [reply packages/theme/src/theme/_forms.scss]: Indeed, that doesn't fix for non webkit browsers :/
I changed it to use `Highlight` color that takes the default highlight color.

File: packages/theme/src/theme/_forms.scss (hunk 2 above)
Comment (jsomsanith) [reply packages/theme/src/theme/_forms.scss]: Done

**Comment by jsomsanith** [comment]
Oh you're right, that's weird that a search dooesn't find anything :/
I found it here : https://ghinda.net/article/mimic-native-focus-css/

**Pull request merged**